Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'plek', '~> 1.10.0'
 gem 'airbrake', '~> 4.2.0'
 
 gem 'gds-sso', '~> 11.0.0'
-gem 'gds-api-adapters', '~> 18.10.0'
+gem 'gds-api-adapters', '~> 20.1.0'
```

Comment:
Your commit says 20.1.1, but this is 20.1.0.  Probably worth pinning to 20.1.1 given 20.1.0 has a bug.